src/packages/solid/components/Key/Key.styles.ts

```diff
@@ -27,24 +30,110 @@ export type KeySizes = {
 
 export type KeySize = keyof KeySizes;
 
-export const styles = {
-  Container: {
+export interface KeyStyles {
+  tone: Tone;
+  Container: NodeStyleSet<{ padding: number[] }>;
+  Text: TextStyleSet;
+}
+
+type KeyStyleProperties = {
```

Comment:
keySpacing, baseWidth, and sizes need to be updated, nodeStyles doesn't contain any of these types